src/proto/robot_status_msg.proto

```diff
@@ -72,10 +75,32 @@ enum ErrorCode
     DRIBBLER_MOTOR_HOT = 4;
 }
 
+enum MotorFault
+{
+    RESET                             = 0;
+    DRIVER_OVERTEMPERATURE_PREWARNING = 1;
```

Comment:
can we link to the trinamic doc with what each is

Reply:
@itsarune you would probably know more about this? i'll try to find docs for this in the meantime

Reply:
updated